unified-scheduler-pool/src/lib.rs

```diff
@@ -850,7 +833,11 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
         );
     }
 
-    fn start_threads(&mut self, context: &SchedulingContext) {
+    fn start_threads(
+        &mut self,
+        context: SchedulingContext,
```

Comment:
all of relevant api now takes ownership of `SchedulingContext` because their impl always consumes. so, there's no reduced `.clone()`s even if we pass it via references.